src/patternfly/components/Accordion/accordion.hbs

```diff
@@ -0,0 +1,6 @@
+<ul class="pf-c-accordion{{#if accordion--modifier}} {{accordion--modifier}}{{/if}}"
```

Comment:
The [wai-aria example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html) uses `<dl>` but when I test their implementation in VO, I don't get any of the semantics associated with `<dl>`. I'm not sure if that's because they change the role of `<dt>` to a heading or not, but doing that seems to render the use of `<dl>` kind of pointless. 

This implementation is also missing the `<dt>` element, and it might be worth checking in a screen reader what happens when you nest all the toggle elements like this:
`<dt><h2><button class="pf-c-accordion__item-button" aria-expanded="false">...`
instead of follow what wai-aria does with 
`<dt role="heading" aria-level="3"><button aria-expanded="true" ...>...`

But I think it would also be fine just use `<div>`s for both `.pf-c-accordion` and `.pf-c-accordion__item` in this implementation, especially if the nesting suggestion above doesn't produce anything more meaningful in the screen reader experience.

Reply:
ya I was wondering why we couldn't use `<dt>` in place of the `<h2>` if we are going with the `<dl>`, `<dd>` structure

Reply:
I updated this to use `<dl>` > `<div>` > `<dt>` > `<dd>`  We have the extra div because of the blue border on the left, but I could try rearranging the CSS to style this border without?

Reply:
ya you might be able to just style the border of the `<dt>` and the expanded `<dd>` rather than having the wrapping div. 🤔

Reply:
`.pf-c-accordion-item` as a `<div>` wrapper takes on `accordion-item--IsExpanded` and unassigns hidden to `.pf-c-accordion-item-expanded-content`. But that is because it is the parent of `.pf-c-accordion-item-expanded-content` . Without this div wrapper, `.pf-c-accordion-item__button` sits adjacent to `.pf-c-accordion-item__expanded-content`and because they are siblings the expanded assignment doesn't work. However the wai-aria example uses js to toggle which item is **hidden**. So it seems to me that it needs this parent wrapper div to handle the expanded content in CSS. @matthewcarleton